ConnectorNavisworks/ConnectorNavisworks/Bindings/ConnectorNavisworksBindings.Filters.cs

```diff
@@ -68,20 +66,31 @@ public override List<ISelectionFilter> GetSelectionFilters()
       filters.Add(savedViewsFilter);
     }
 
-    var clashPlugin = _doc.GetClash();
-    var clashTests = clashPlugin.TestsData;
-
-    var groupedClashResults = clashTests?.Tests.Select(GetClashTestResults).Where(x => x != null).ToList();
+    try
+    {
+      var clashPlugin = _doc.GetClash();
+      var clashTests = clashPlugin?.TestsData;
 
-    if (groupedClashResults?.Count >= 0)
+      if (clashTests != null)
+      {
+        var groupedClashResults = clashTests.Tests.Select(GetClashTestResults).Where(x => x != null).ToList();
+
+        if (groupedClashResults.Count >= 0)
+        {
+          //  var clashReportFilter = new TreeSelectionFilter
+          //  {
+          //    Slug = "clashes", Name = "Clash Detective Results", Icon = "MessageAlert",
+          //    Description = "Select group clash test results.",
+          //    Values = groupedClashResults
+          //  };
+          //  filters.Add(clashReportFilter);
+        }
+        Console.WriteLine("Sending Clashes is not supported yet.");
```

Comment:
I guess this should come out or go via structured logger?

Reply:
i only added this to clear the IDE complaining of an empty conditional branch